stencil_main.py

```diff
@@ -147,17 +105,19 @@ def main(
     nx,
     ny,
     nz,
-    backend,
     stencil_name,
+    backend,
+    numba_parallel=False,
+    gt4py_backend="numpy",
     num_iter=1,
     num_halo=2,
     plot_result=False,
     create_field=True,
     field_name="test",
     df_name="df",
-    save_runtime=False
+    save_runtime=False,
 ):
-    """Driver for high-level comparison of stencil computation. HPC4WC group 7 coursework."""

```

Comment:
You can omit the second sentence - it's not necessary to understand the scope of the function.